File: python/phonebook.py
people = {
    "name": "Bola", "number":"+234-912345678",
    "name": "Tunde", "number":"+234-812345678"
}

name = input("name: ")

if name in people:
    print(f"Number: {people[name]}")
else:
    print("Not found")

File: Introduction-to-computer-science/algorithms/phonebook.c
#include "cs50.h"
#include <stdio.h>
#include <string.h>

typedef struct 
{
    string name;
    string number;
} person;


int main (void) {
    person people[1];

    people[0].name = "Olusegun";
    people[0].number = "09025768559";

    people[1].name = "Anna";
    people[1].number = "89047553561";
}

